src/libraries/Microsoft.PowerFx.Connectors/Microsoft.PowerFx.Connectors.csproj

```diff
@@ -32,8 +33,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.OpenApi.Readers" Version="1.3.1" />
     <PackageReference Include="System.Text.Json" Version="5.0.2" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="System.Memory" Version="4.5.4" />
+    <PackageReference Include="System.Collections.Immutable" Version="6.0.0" />
```

Comment:
This change can break consumers.  When building Dataverse.EVal against this private change, I get this error:

Severity	Code	Description	Project	File	Line	Suppression State
Error	NU1605	Detected package downgrade: System.Collections.Immutable from 6.0.0 to 5.0.0. Reference the package directly from the project to select a different version. 
 Microsoft.PowerFx.Dataverse.Eval -> Microsoft.PowerFx.Core 0.2.3-preview.20221219-004 -> System.Collections.Immutable (>= 6.0.0) 
 Microsoft.PowerFx.Dataverse.Eval -> System.Collections.Immutable (>= 5.0.0)	Microsoft.PowerFx.Dataverse.Eval	D:\dev\temp\Power-Fx-Dataverse\src\PowerFx.Dataverse.Eval\Microsoft.PowerFx.Dataverse.Eval.csproj	1

Reply:
Agreed. I've added a comment in this PR